proxy.go

```diff
@@ -241,14 +241,22 @@ func executeWithRetry(
 			// comment s.host.dec() line to avoid double increment; issue #322
 			// s.host.dec()
```

Comment:
@sigua-cs should I remove this comment as I add the decrement in line 255?

Reply:
yes you should since your PR fix this legacy quick & dirty (and broken) fix.

Reply:
done